mmdeploy/apis/utils/calibration.py

```diff
@@ -46,6 +43,10 @@ def create_calib_input_data(calib_file: str,
             'val', defaults to 'val'.
         device (str): Specifying the device to run on, defaults to 'cpu'.
     """
+    import h5py
+    import tqdm
```

Comment:
does it mean `tqdm` and `h5py` should be moved to optional.txt in requirements?

Reply:
Reasonable.